third_party/fmt/include/fmt/format-inl.h

```diff
@@ -405,7 +405,7 @@ class fp {
     assign(d);
     constexpr int min_normal_e = std::numeric_limits<float>::min_exponent -
                                  std::numeric_limits<double>::digits;
-    significand_type half_ulp = 1 << (std::numeric_limits<double>::digits -
+    significand_type half_ulp = (uint64_t)1 << (std::numeric_limits<double>::digits -
```

Comment:
`std::numeric_limits<double>::digits == 53` which causes undefined behavior if applied to a `int32_t`
So this has to be cast

Reply:
Ah, it's (53 - 24 - 1) which is equal to 28, so this cast is not needed 😅